new-frontiers-14/frontier-station-14
# Localize docks

## About the PR



Localize docks.

## Why / Balance


💪 :ru:

Localization strings are looked up on the server side at the moment - thought was that we can send less data per dock set with shorter strings.  If this needs to change, that can move to the dock controls themselves to push this clientside.

## How to test


Run game, check the docks at Frontier, the Expeditionary Lodge, the Trade Outpost, the NFSD outpost, and McCargo (DARIN)

## Media


Holy crap, it's the same!
![image](https://github.com/user-attachments/assets/8150321c-ec17-4f12-9a4a-9577fa8f100d)

## Requirements



## Breaking changes


Localized strings will need to be added for other languages, the dock labels will be very long otherwise.

**Changelog**


:cl:
- add: Localized dock names.
